### ISSUE TITLE
Modeling

### ISSUE THREAD

**Issue by msaivepinkall** [JeromeSauer/Capstone_WindPowerPredicting#11]
- [x] Error Analysis
- [x] Cross validation
- [x]  Hyperparameter tuning
- [x]  Significance test
- [x] possible models:

-- LightGBM
-- eXtreme Gradient Boosting (xgBoost)
-- Gradient Boosting Machines organized in 2 layers
-- Gradient Boosting Decision Trees
-- Quantile Regression Forest
-- Knn
-- Multiple Quantile Regression
-- Kernel Density Estimation
-- Randomforest + feature importance
-- NN 
-- Polynomial regression
-- Support Vector Machine for regression with Linear Kernel (SVR-Linear)
-- SVR RBF
-- Multi-Layer Perceptron with multiple layers (MLP) and K-Nearest Neighbors (K-NN)
